src/status_im/ui/screens/accounts/recover/events.cljs

```diff
@@ -41,7 +41,8 @@
                    :updates-private-key private
                    :mnemonic            ""
                    :signed-up?          true
-                   :signing-phrase      phrase}]
+                   :signing-phrase      phrase
+                   :settings            {:wallet {:visible-tokens {:testnet #{:STT} :mainnet #{:SNT}}}}}]
```

Comment:
Can we make sure this is not duplicated with regular account creation?

Reply:
@jeluard moved to constants, please check!